ignite/templates/app/files/app/ibc.go.plush

```diff
@@ -162,16 +165,27 @@ func (app *App) registerIBCModules() {
 	app.ScopedICAHostKeeper = scopedICAHostKeeper
 	app.ScopedICAControllerKeeper = scopedICAControllerKeeper
 
-    // register IBC modules
+	// register additional types
+	ibctm.AppModuleBasic{}.RegisterInterfaces(app.interfaceRegistry)
+	solomachine.AppModuleBasic{}.RegisterInterfaces(app.interfaceRegistry)
+
+	// register IBC modules
 	if err := app.RegisterModules(
 		ibc.NewAppModule(app.IBCKeeper),
 		ibctransfer.NewAppModule(app.TransferKeeper),
 		ibcfee.NewAppModule(app.IBCFeeKeeper),
 		icamodule.NewAppModule(&app.ICAControllerKeeper, &app.ICAHostKeeper),
 		capability.NewAppModule(app.appCodec, *app.CapabilityKeeper, false),
-		ibctm.AppModule{},
```

Comment:
Any reason of this change? `RegisterModules` will call all the necessary methods: https://github.com/cosmos/cosmos-sdk/blob/721dec9b608983d5e482f06706c7d23320c8701c/runtime/app.go#L63